### ISSUE TITLE
BEM Assignment

### ISSUE THREAD

**Issue by CCucalonUTA** [KV/CTEC-3350#5]
Surface review of the assignment:

* There are no styles for this page
* The image gallery is using a table as its structure, however, using a table will not work on a mobile device, I would encourage you to update this component. Also, just as my example I would like to see how the code renders on the page besides the wireframe picture you added.
* I'm also looking at your Wireframe and these components are not in them, please either update the components or the Wireframe

Please provide fixes to these issues before 10 p.m. Monday April 10th, 2017.

Thank you.

**Comment by KV** [comment]
I made those adjustments

**Comment by CCucalonUTA** [comment]
* The styles still need work.
* The path to the images is not correct
* The organization of the site is hard to follow as I don't know where one section begins and ends
* Please make sure your code and HTML rendering looks like your mock-up
* There are two components that are not rendered on the page
